services/server/src/library/CoordinateValidation.ts

```diff
@@ -0,0 +1,24 @@
+import _ from "lodash";
+
+const countDecimals = (n: number) => {
+    if ((n % 1) != 0) {
+        return n.toString().split(".")[1].length;
+    }
```

Comment:
What does the number of decimal places have to do with if it is a valid coordinate? Also comparing a float using a remainder thing sounds sus and finicky. Also converting to string and spliting by decimal place also sounds sketchy.

Reply:
I agree. I felt sus while making it

Reply:
Did some research. I think the .toString() is the javascriptonic way. ChatGPT generated this 

```ts
function countDecimalPlaces(num: number): number {
  let decimalCount = 0;
  let numCopy = num;

  while (numCopy % 1 !== 0) {
    decimalCount++;
    numCopy *= 10;
  }

  return decimalCount;
}
```

Like the original implementation more, feels cleaner to me